src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -17,42 +17,79 @@
 import wolox.training.models.Book;
 import wolox.training.repositories.BookRepository;
 
+/**
+ * @author josefranciscodelvecchio
+ * Controller for CRUD Book
+ */
 @RestController
 @RequestMapping("/api/books")
 public class BookController {
 
     @Autowired
     private BookRepository bookRepository;
 
+    /**
+     * This method find all Books
+     * @return {@link Book}
```

Comment:
Aquí faltó el enlace a `List`
```suggestion
     * @return {@link List}<{@link Book}>
```